content/projects/gsoc/students.adoc

```diff
@@ -87,63 +292,23 @@ between students, org admins and mentors.
 
 See the link:/projects/gsoc/#office-hours[main GSoC page] for the schedule.
 
-== Expectations from students
-
-NOTE: The section below is under development. 
-The expectations may slightly change before the beginning of the Student application period.
-
-=== Student application process
+== Post-GSoC
 
-. We expect students to get involved into project discussions on the beginning of the student application period in order to have opportunity to discuss the project with them and to jointly review the proposal drafts.
-. We expect students to attend at least one office hours during the application period.
-. We expect proposals to contain all the sections discussed in the link:http://write.flossmanuals.net/gsocstudentguide/what-is-google-summer-of-code/[GSoC Student Guide]
-
-=== Community bonding
-
-Students and mentors are expected to...
+Congratulations, you have made it to the end!
```

Comment:
Should be an easter egg here :)

Reply:
Er... I know what an easter egg is but I am not sure if you want me to do something here.